catanatron_core/catanatron/game.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         vps_to_win: int = 10,
         catan_map: CatanMap = None,
         initialize: bool = True,
+        disable_accumulator_copy = True,
```

Comment:
I think we can remove the configuration all together to keep this __init__ simple. Let's just not copy, and add documentation to the `class GameAccumulator` reminder implementors to not mutate game otherwise we are not responsible... haha :D :) 

It should also simplify the later code too.

Reply:
Sounds good! 👍